feat: add `Quick start` section and remove tedious parts in README.md (#490)
nervosnetwork/ckb-sdk-java: README.md

```

### Sign and send transaction
To send transaction you build to CKB network, you need to
1. sign transaction with your private key.
2. send signed transaction to CKB node, and wait it to be confirmed.

```java
// Assume that you already have an unsigned transaction named `rawTx`
Secp256k1SighashAllBuilder signBuilder = new Secp256k1SighashAllBuilder(rawTx);

// Prepare private key to sign
List<ScriptGroupWithPrivateKeys> scriptGroupWithPrivateKeysList = new ArrayList<>();
scriptGroupWithPrivateKeysList.add(
    new ScriptGroupWithPrivateKeys(
        new ScriptGroup(Arrays.asList(0, 1, 2)),  // indices of input cells that are to be unlocked by this private key
        Arrays.asList("e79f3207ea4980b7fed79956d5934249ceac4751a4fae01a0f7c4a96884bc4e3")  // your private key
    )
);

// Sign transaction with private key
for (ScriptGroupWithPrivateKeys scriptGroupWithPrivateKeys : scriptGroupWithPrivateKeysList) {
    signBuilder.sign(scriptGroupWithPrivateKeys.scriptGroup, 
        scriptGroupWithPrivateKeys.privateKeys.get(0));
}

// Send transaction
Transaction tx = signBuilder.buildTx();
ckbApi.sendTransaction(tx);;
```

### Generate a new address

> CKB address support short and full address for the same lock script, but the short address is about to be deprecated and full address is recommended. For more please refer to [CKB rfc 0021](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0021-ckb-address-format/0021-ckb-address-format.md).

Reviewer: ```suggestion
CKB address support short and full address for the same lock script. For more please refer to [CKB rfc 0021](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0021-ckb-address-format/0021-ckb-address-format.md).
```

Generate short address (`secp256k1_blake160`)
```java
String shortAddress = AddressTools.generateShortAddress(network).address;
```

Generate full address (`secp256k1_blake160`)
```java
String shortAddress = AddressTools.generateShortAddress(network).address;
Script script = AddressTools.parse(shortAddress).script;
String fullAddress = AddressGenerator.generateBech32mFullAddress(network, script);
```

### Parse and validate address

```java
AddressParseResult parseResult = AddressParser.parse("ckt1qyqz9r9w9gkf5799a477jx07kltx6qqgxv8qn492h3");
System.out.println("address info - network: " + parseResult.network + ", script: " + parseResult.script + ", type: " + parseResult.type);

// Blow will throw `AddressFormatException` since the input is not a valid address
AddressParser.parse("ckt1qyqz9r9w9gkf5799a497jx07kltx6qqgxv8qn492h3");
```

### Mercury

> [Mercury](https://github.com/nervosnetwork/mercury) is a development service in CKB ecosystem, providing many useful [RPC APIs](https://github.com/nervosnetwork/mercury/blob/main/core/rpc/README.md) for development like querying transaction or getting udt asset information. You need to deploy your own mercury and sync data with the network before using it.

Reviewer: ```suggestion
[Mercury](https://github.com/nervosnetwork/mercury) is a development service in CKB ecosystem, providing many useful [RPC APIs](https://github.com/nervosnetwork/mercury/blob/main/core/rpc/README.md) for development like getting balance or building transfer transaction. You need to deploy your own mercury and sync data with the network before using it.
```

### Setup
ckb-java-sdk provides a convenient client to help you easily interact with [CKB](https://github.com/nervosnetwork/ckb), [CKB-indexer](https://github.com/nervosnetwork/ckb-indexer) or [Mercury](https://github.com/nervosnetwork/mercury) node.

```java
// Set up client. If you do not use ones of these node, just set them to null;
String ckbUrl = "http://127.0.0.1:8114";
String indexerUrl = "http://127.0.0.1:8114";
String mercuryUrl = "http://127.0.0.1:8116";
DefaultCkbApi ckbApi = new DefaultCkbApi(ckbUrl, mercuryUrl, indexerUrl, false);
```

You can leverage this client to call any RPC APIs provided by CKB, CKB-indexer or Mercury in Java code.
```java
Block block = ckbApi.getBlock("0x77fdd22f6ae8a717de9ae2b128834e9b2a1424378b5fc95606ba017aab5fed75");
```

For more about RPC APIs, please check: 
- [CKB RPC doc](https://github.com/nervosnetwork/ckb/blob/develop/rpc/README.md)
- [CKB-indexer RPC doc](https://github.com/nervosnetwork/ckb-indexer/blob/master/README.md)
- [Mercury RPC doc](https://github.com/nervosnetwork/mercury/blob/main/core/rpc/README.md).

### Build transaction

> 

Reviewer: Please add mercury guidelines here

> [Mercury](https://github.com/nervosnetwork/mercury) is a development service in CKB ecosystem, providing many useful [RPC APIs](https://github.com/nervosnetwork/mercury/blob/main/core/rpc/README.md) for development like querying transaction or getting udt asset information. You need to deploy your own mercury and sync data with the network before using it.

Reviewer: ```suggestion
[Mercury](https://github.com/nervosnetwork/mercury) is a development service in CKB ecosystem, providing many useful [RPC APIs](https://github.com/nervosnetwork/mercury/blob/main/core/rpc/README.md) for development like getting the balance of giving assets, building transfer transactions, query_transactions and so on. You need to deploy your own mercury and sync data with the network before using it.
```

// Find live cells and calculate capacity balance with the help of CKB indexer.
BigInteger feeRate = BigInteger.valueOf(1024);
List<String> SendAddresses = Arrays.asList("ckt1qyqrdsefa43s6m882pcj53m4gdnj4k440axqswmu83");
CollectResult collectResult = txUtils.collectInputs(SendAddresses, txBuilder.buildTx(), feeRate, Sign.SIGN_LENGTH * 2);

// Generate cell outputs - the target address.
List<CellOutput> cellOutputs = txUtils.generateOutputs(receivers, changeAddress);
txBuilder.addOutputs(cellOutputs);
// Charge back (if inputs capacity - fee > outputs capacity)
if (Numeric.toBigInt(collectResult.changeCapacity).compareTo(BigInteger.ZERO) > 0) {
    cellOutputs.get(cellOutputs.size() - 1).capacity = collectResult.changeCapacity;
    txBuilder.setOutputs(cellOutputs);
}

// Add witnesses placeholder for latter signature
for (CellsWithAddress cellsWithAddress : collectResult.cellsWithAddresses) {
    txBuilder.addInputs(cellsWithAddress.inputs);
    for (int i = 0; i < cellsWithAddress.inputs.size(); i++) {
        txBuilder.addWitness(i == 0 ? new Witness(Witness.SIGNATURE_PLACEHOLDER) : "0x");
    }
}
// Build the unsiged transaction
Transaction rawTx = txBuilder.buildTx();
```

> A more recommended way is to directly call RPC API `build_simple_transfer_transaction` provided by Mercury, which could help you do almost everything above in building transaction.

Reviewer: ```suggestion
A more recommended way is to directly call RPC API `build_transfer_transaction` or `build_simple_transfer_transaction` provided by Mercury, which could help you do almost everything above in building transaction.
```
